### ISSUE TITLE
Train BERT SQUAD fine-tuning task to INT8 

### ISSUE THREAD

**Issue by rnaidu02** [mlperf/inference_policies#127]
On Intel CPU
 - post-training quantization to INT8 has accuracy issues and we had issues SSD models during v0.5
 - symmetrically quantized int8 finetuned models are having accuracy issues

for the above reasons, Intel would like to train fine-tuning using Huggingface+NNCF plugin to produce symmetric weights and asymmetric activations.

**Comment by DilipSequeira** [comment]
NVIDIA is planning to contribute an ONNX model using per-tensor symmetric INT8 weights and activations that has accuracy close to the reference. So that may obviate this issue.

But you should feel free to contribute your own quantized model too. Two provisos:

-  you should fine-tune starting from the MLPerf reference weights
- as per https://docs.google.com/document/d/1YYgB_UR7jE5rQFMHet6m87s9qaVwYcqCuHPkBrRZgNs, NVIDIA, as model owner, is responsible for ensuring that third party contributions are integrated into loadgen, and we would request that Intel do this.
